Embedded Metadata.js

```diff
@@ -934,12 +934,11 @@ function relativeToAbsolute(doc, url) {
 	if (url[0] == '/') {
 		// relative to root
 		return doc.location.protocol + '//' + doc.location.host
-			+ (doc.location.port.length ? ':' + doc.location.port : '')
```

Comment:
Was this part on purpose?

Reply:
Yeah. `Location.host` [already includes the port if necessary](https://developer.mozilla.org/en-US/docs/Web/API/Location/host). `Location.hostname` doesn't. I'm not sure why the code I ripped for this method was adding it again.